src/components/Login/EmailVerification/VerificationResult.tsx

```diff
@@ -48,7 +48,7 @@ const VerificationResult: React.FC<Props> = (props: Props) => {
           width="300px"
           marginTop="20px"
           onClick={verified ? handleCloseModal : handlePrevStep}
-          rightIcon={<RiArrowRightLine />}
+          // rightIcon={<RiArrowRightLine />}
```

Comment:
If it was left on purpose, leave a TODO here. Otherwise, delete it.

Reply:
That's a minor UI refinement; I've deleted it :)